src/flatpickr.js

```diff
@@ -1801,8 +1801,9 @@ function Flatpickr(element, config) {
 				.map(dObj => self.formatDate(dObj, self.config.altFormat))
 				.join(joinChar);
 		}
-
-		triggerEvent("ValueUpdate");
+		if (triggerChange !== false) {
```

Comment:
Maybe this is unnecessary. I think that ``` triggerChange === false ``` must disable all events triggering. Why only onChange?